qubes/dochelpers.py

```diff
@@ -41,12 +41,17 @@
 import sphinx.errors
 import sphinx.locale
 import sphinx.util.docfields
+from sphinx.util import logging
 
 import qubes.tools
 
 SUBCOMMANDS_TITLE = 'COMMANDS'
 OPTIONS_TITLE = 'OPTIONS'
 
+try:
+    log = logging.getLogger(__name__)
+except:
```

Comment:
pylint don't like bare except and I don't like it either. Please use `except AttributeError`, same as in core-admin-client.